### PR TITLE
Pinnako: farm integration

### PR DESCRIPTION
**Pull request by Xiaxuliang**
- **Project Name**: Pinnako

- **Project Website**:  https://pinnako.io/

- **Description**:  Pinnako stands as a pioneering decentralized perpetual trading platform, offering a diverse array of markets bolstered by substantial liquidity.

### CODE REVIEW

File: apps/workshop/projects/Pinnako/modules/ExampleFarm.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            apr: { year: stats.apr },
+          },
+        ],
+      },
+    ];
+    return vaults;
+  },
+
+  async fetchUserPositions() {
+    // TODO: Fetch User Positions
```
Comment (artembondadefiyeld) [inline apps/workshop/projects/Pinnako/modules/ExampleFarm.ts:42]: do you have plans to add this?

Comment (Xiaxuliang) [reply apps/workshop/projects/Pinnako/modules/ExampleFarm.ts]: already added

File: apps/workshop/projects/Pinnako/modules/ExampleFarm.ts
Comment (witem) [inline apps/workshop/projects/Pinnako/modules/ExampleFarm.ts:1]: Is this example farm, or it's possible to use some better name?

File: apps/workshop/projects/Pinnako/tests/project.spec.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  beforeEach(async (context) => {
+    context.project = await createTestProject({
+      name: 'Pinnako',
+      path: join(__dirname, '../index.ts'),
+      // modules: [/** Your Module Here **/],
+      contracts: mockContracts,
+    });
+  });
+
+  test.todo('Fetches all the expected tokens');
```
Comment (witem) [inline apps/workshop/projects/Pinnako/tests/project.spec.ts:21]: Do you planning to fill this TODOs?

**Comment by Xiaxuliang** [comment]
> Is this the correct APR? image
> 
> jfyi https://development.de.fi/explore?platformIds=178

yes is correct

File: apps/workshop/projects/Pinnako/tests/project.spec.ts (hunk 2 above)
Comment (Xiaxuliang) [reply apps/workshop/projects/Pinnako/tests/project.spec.ts]: no plan, i delete it

File: apps/workshop/projects/Pinnako/modules/ExampleFarm.ts
Comment (Xiaxuliang) [reply apps/workshop/projects/Pinnako/modules/ExampleFarm.ts:1]: has changed

**Review by witem — APPROVED** [review]
lgtm

**Pull request merged**